Aplib.Core/Belief/IBeliefSet.cs

```diff
@@ -5,9 +5,9 @@
     /// </summary>
     public interface IBeliefSet
     {
-      /// <summary>
-      /// Updates all beliefs in the belief set.
-      /// </summary>
-      void UpdateBeliefs();
+        /// <summary>
+        /// Updates all beliefs in the belief set.
+        /// </summary>
+        void UpdateBeliefs();
```

Comment:
Undo these changes, as they are already made in 'feat: add MemoryBelief' and they don't really fit in this PR